src/guide/style.md

```diff
@@ -2,13 +2,93 @@
 
 This is a work-in-progress guide to writing and formatting content for LaunchCode education programs.
 
+For general grammar, style, and punctuation guidelines not covered here, refer to the [Purdue Online Writing Lab (OWL)](https://owl.purdue.edu/owl/general_writing/index.html).
+
 ## Headings
 
 For headings, use sentence case (all major words of the headline capitalized). For the definition of "major word" we defer to AP style:
 
 > Capitalize the first word of every letter except articles, coordinating conjunctions, and prepositions of three letters or fewer. There’s one exception: Any word that is the first word in the headline or the last word should be capitalized, regardless of its part of speech.
```

Comment:
So, This not That, Neither but not nor, Either Twas or Twasn't, but not for, Yet?